### DISCUSSION TITLE
How to draw shapes on no_std Mcu?

### DISCUSSION THREAD

**Discussion by yanshay** [slint-ui/slint#4393] [Q&A]
`Path` (both 'svg' string and the commands) is not supported on `no_std', how can I draw shapes like triangles and others, including filling them? 
Preferably also want to color them with gradient if possible.

**Comment by ghost** [comment]
Unfortunately that is not possible in the moment. There is an issue to track the progress of path implementation for the software renderer: https://github.com/slint-ui/slint/issues/4178

Alternatively you could draw it to an image and display it in Slint.
